.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,66 @@
+name: Feature Request
+description: Suggest an idea for NextAuth.js
+title: "[Feature Request]: "
+labels: enhancement
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for reaching out to us regarding the awesome feature that you believe should be included in the NextAuth.js library. Please provide the following information:
+  - type: input
+    id: summary
+    attributes:
+      label: Summary 💭
+      description: A clear and concise summary of the feature being proposed
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description 📓
+      description: Please provide a more in-depth description of the feature proposed.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Make sure you provide plenty of [links]() to external documentation and inline code examples like so:
+
+        ```js
+        function myAwesomeNextAuthFeature() {
+          return 💚
+        }
+        ```
+
+        Take time thinking about what you want to say and help us understand your proposal making sure that this description contains:
+
+        - **purpose of the feature**
+        - **potential problems**
+        - **potential alternatives**
```

Comment:
Should we also turn this into comments as well? That information is going to be a bit noisy for maintainers in my opinion. What do you think?

Reply:
Yes, I also think it serves better as a comment, which would only be visible while filing the issue.

Reply:
I just noticed in the [documentation for Github Issue Forms YAML Syntax](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#markdown):

> You can use a markdown element to display Markdown in your form that provides extra context to the user, but is not submitted.